QUEST.tsv

```diff
@@ -2777,67 +2777,67 @@ QUEST_20151016_002784
 QUEST_20151016_002785	
 QUEST_20151016_002786	
 QUEST_20151016_002787	
-QUEST_20151016_002788	
+QUEST_20151016_002788	Ferret
 QUEST_20151016_002789	{memo X}
 QUEST_20151016_002790	{memo X}
 QUEST_20151016_002791	{memo X}
-QUEST_20151016_002792	
-QUEST_20151016_002793	
-QUEST_20151016_002794	
-QUEST_20151016_002795	
-QUEST_20151016_002796	
+QUEST_20151016_002792	Freed girl has ran away somewhere with urgency.
+QUEST_20151016_002793	Girl looked at my way as if she wanted me to follow her.
```

Comment:
The girl looked at me as if she wanted me to follow her.
The girl stared at me as if she wanted me to follow her.

?